# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/CrossShadowBoundaryRange.h
# Commit: 5480477f3599
# Full Hash: 5480477f35993b93846c693c8e388f727ea18dc4
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-06-21 04:13:05
# Regressor Bug: 1881096
# File Overlap Count: 2
# Description:
#   Bug 1881096 - Add basic support of character removal to CrossShadowBoundaryRange r=emilio,jjaschke,dom-core
#   
#   Because nsPrintJob would remove characters from the selected boundary
#   nodes, the offsets of the boundaries needs to be updated if this
#   happens.
# ==============================================================================

diff -r 2d732ba0d3b6 -r 5480477f3599 dom/base/CrossShadowBoundaryRange.h
--- a/dom/base/CrossShadowBoundaryRange.h	Thu Jun 20 20:15:50 2024 +0000
+++ b/dom/base/CrossShadowBoundaryRange.h	Thu Jun 20 20:15:51 2024 +0000
@@ -56,6 +56,7 @@
     return StaticRange::SetStartAndEnd(aStartBoundary, aEndBoundary);
   }
 
+  NS_DECL_NSIMUTATIONOBSERVER_CHARACTERDATACHANGED
   NS_DECL_NSIMUTATIONOBSERVER_CONTENTREMOVED
 
  private: